keps/sig-network/563-dual-stack/README.md

```diff
@@ -1362,23 +1356,25 @@ This capability will move to stable when the following criteria have been met.
   the enablement)?**
   Yes.
 
+  [TODO] If we disable the featuregate, do I also have to remove the CLI parameters or are those still accepted?
+
 * **What happens if we reenable the feature if it was previously rolled back?**
   Similar to enable it the first time on a cluster.
 
+  [TODO] How to coordinate the enablement and re-enablement
```

Comment:
Feedback brought up this question and we could clarify here.

Reply:
Full feedback:
 _Is it possible to get an imbalance between IPv4 and IPv6 endpoints? Or do we expect all endpoints to be dual stack and therefore there are identical numbers for each family for a given service? Thinking about the re-enablement case, here, for example. If we have a service with 10,000 endpoints, all IPv4, and we re-enable in the control plane such that the IPv6 clusterIP is allocated, but we now need to roll out the kubelet/kube-proxy enablement and create dual stack pods, we're going to have a long period where we have far more IPv4 endpoints than IPv6...but DNS will return both addresses equally (actually anecdotally it seems AAAA come back faster) so we may crush those few endpoints..._

Reply:
We don't load balance across families, and with no selectors we don't get endpoints.

Reply:
If you feature-flag off dual-stack, we do not edit your service. If you disable dual-stack from the controller manager, the service will get single-stack endpoints.